.env

```
@@ -9,7 +9,7 @@ _APP_CONSOLE_WHITELIST_EMAILS=
 _APP_CONSOLE_SESSION_ALERTS=enabled
 _APP_CONSOLE_WHITELIST_IPS=
 _APP_CONSOLE_COUNTRIES_DENYLIST=AQ
-_APP_CONSOLE_HOSTNAMES=localhost,appwrite.io,*.appwrite.io
+_APP_CONSOLE_HOSTNAMES=localhost,appwrite.io,*.appwrite.io,posted-costumes-alphabetical-census.trycloudflare.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The _APP_CONSOLE_HOSTNAMES key should go before the _APP_CONSOLE_SESSION_ALERTS key

(UnorderedKey)